### ISSUE TITLE
tinyMCE/ace not restored after Turbo page transition

### ISSUE THREAD

**Issue by m-vo** [contao/contao#7391]
### Affected version(s)

5.4

### Description

With the changes from #7387 there now are two issues after saving a record with a tinyMCE/ace field:

* The tinyMCE Editor is gone (gets destroyed somehow/not reinitialized).
* The ace editor loses its size and only restores it after clicking into it.
 
Sidenote: For ace, we're currently using some rather complicated init stuff to make it use its full height. This could probably simplified to something like that (maybe already fixing the issue):
```js
const editor = ace.edit(element, {
    theme: document.documentElement.dataset.colorScheme === 'dark' ? 'ace/theme/twilight' : 'ace/theme/clouds',
    mode: 'ace/mode/foo',
    maxLines: 100, // >= lines from content
    wrap: true,
    useSoftTabs: false,
    autoScrollEditorIntoView: true,
});

editor.container.style.lineHeight = '1.45';
whitespace.detectIndentation(editor.getSession());
``` 

/cc @zoglo

**Comment by aschempp** [comment]
is this fixed by https://github.com/contao/contao/pull/7394?

**Comment by m-vo** [comment]
Ultimately fixed by #7408.
